File: rust/tcl-registry/src/commands/iapps/tmsh__display_threshold.rs
//! `tmsh::display_threshold` command.
use crate::prelude::*;
const FORMS: &[FormSpec] = &[FormSpec {
    kind: FormKind::Default,
    synopsis: "tmsh::display_threshold ?value?",
}];

pub fn spec() -> CommandSpec {
    CommandSpec {
        name: "tmsh::display_threshold",
        dialects: Some(DialectSet::IAPPS),
        arity: Arity::at_least(0),
        hover: Some(HoverSnippet::brief(
            "Re-enables a display-threshold in the script.",
            &["tmsh::display_threshold ?value?"],
            "F5",
        )),
        forms: FORMS,
        ..CommandSpec::DEFAULT
    }
}

File: core/commands/registry/iapps/tmsh__display_threshold.py
"""tmsh::display_threshold -- F5 TMSH logging and display command."""

from __future__ import annotations

from .._base import CommandDef
from ..models import CommandSpec
from ._base import _tmsh_spec, register


@register
class TmshDisplayThreshold(CommandDef):
    name = "tmsh::display_threshold"

    @classmethod
    def spec(cls) -> CommandSpec:
        return _tmsh_spec(
            "tmsh::display_threshold",
            "Re-enables a display-threshold in the script.",
            "tmsh::display_threshold ?value?",
        )
